benchmarks/benchmark.js

```diff
@@ -318,6 +319,16 @@ if (process.env.PORT) {
       }).catch(console.log)
     })
   }
+
+  experiments.superagent = () => {
+    return makeParallelRequests(resolve => {
+      superagent.get(dest.url).pipe(new Writable({
```

Comment:
You should be creating and setting and agent, otherwise the benchmark would not be even

Reply:
@mcollina 
Superagent doesn't expose a verbose API for setting such agent, so I used a workaround.